libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -3572,6 +3632,33 @@ bool AP_AHRS::using_extnav_for_yaw(void) const
     return false;
 }
 
+// check if using gps
+bool AP_AHRS::using_gps(void) const
+{
+    switch (active_EKF_type()) {
+#if HAL_NAVEKF2_AVAILABLE
+    case EKFType::TWO:
+        return EKF2.using_gps();
+#endif
+#if AP_AHRS_DCM_ENABLED
+    case EKFType::DCM:
```

Comment:
Should there be a check for DCM?  From a quick look it seems like AP_AHRS_DCM has a _gps_use member.